api/balancehelper.py

```diff
@@ -29,7 +29,7 @@ def get_balancedata(address):
                              atx.propertyid) s2
                         on s1.propertyid=s2.propertyid) f1
                      inner join smartproperties sp
-                     on f1.propertyid=sp.propertyid and (sp.protocol='Mastercoin' or sp.protocol='Bitcoin')
+                     on f1.propertyid=sp.propertyid and (sp.protocol='Omni' or sp.protocol='Bitcoin')
```

Comment:
@Nevtep lets not remove the old mastercoin from this query, instead append Omni as another 'or' statement

sp.protocol='Omni' or sp.protocol='Mastercoin' or sp.protocol='Bitcoin'